src/BlockEntities/CommandBlockEntity.cpp

```diff
@@ -175,21 +180,30 @@ void cCommandBlockEntity::Execute()
 		{
 			// Overwrite field
 			m_CmdBlock->SetLastOutput(cClientHandle::FormatChatPrefix(m_CmdBlock->GetWorld()->ShouldUseChatPrefixes(), "SUCCESS", cChatColor::Green, cChatColor::White) + a_Text);
+			m_CmdBlock->GetWorld()->BroadcastBlockEntity(m_CmdBlock->GetPos());
 		}
 	} CmdBlockOutCb(this);
 
+	AString RealCommand = m_Command;
+
+	// Remove trailing slash if it exists, since console commands don't use them
```

Comment:
The first character can never be a _trailing_ slash, it may only be _leading_.

Reply:
Oops, looks like I reverted my comment to a previous version. Will fix.